README.md

```diff
@@ -2,50 +2,46 @@
+We hold project meetups. Our usual meeting spot is at Industrious (where CandiDate is located). The address is [230 S Broad St, Floor 17, Philadelphia](https://goo.gl/maps/Anoo4SUE9At "Google Maps").
 
-### July 26
+| 📅 Date | ⌚ Time | 🗺 Location | ℹ️ Meetup Details | 💰 Sponsor |
+|--------|---------|-----------|-----------|-------------|
+| Monday, September 19, 2016 | 6:00 PM | CandiDate | [DataPhilly](https://www.meetup.com/DataPhilly/events/234124464/) | Penn Institute for Biomedical Informatics |
+| ~~Tuesday, September 6, 2016~~ | 6:00 PM | CandiDate | [DataPhilly](https://www.meetup.com/DataPhilly/events/233403039/) | Penn Institute for Biomedical Informatics |
+| ~~Tuesday, August 23, 2016~~ | 6:00 PM | CandiDate | [DataPhilly](https://www.meetup.com/DataPhilly/events/233403001/) | Penn Institute for Biomedical Informatics |
+| ~~Tuesday, August 9, 2016~~ | 6:00 PM | CandiDate | [DataPhilly](https://www.meetup.com/DataPhilly/events/233070705/) | Penn Institute for Biomedical Informatics |
+| ~~Tuesday, July 26, 2016~~ | 6:00 PM | CandiDate | [DataPhilly](https://www.meetup.com/DataPhilly/events/232785717/) | Penn Institute for Biomedical Informatics |
+| ~~Tuesday, July 19, 2016~~ | 6:00 PM | CandiDate | [DataPhilly](https://www.meetup.com/DataPhilly/events/232591931/) | [Penn Institute for Biomedical Informatics](http://upibi.org/) |
+| ~~Tuesday, July 12, 2016~~ | 6:00 PM | CandiDate | [DataPhilly](https://www.meetup.com/DataPhilly/events/232470548/) | MilkBoy |
+| ~~Tuesday, July 5, 2016~~ | 6:00 PM | [CandiDate](http://www.candidatephilly.com/) | [DataPhilly](https://www.meetup.com/DataPhilly/events/232137908/) | [Neo Technology](https://neo4j.com/) |
+| ~~Tuesday, June 28, 2016~~ | 6:00 PM | MilkBoy | [DataPhilly](https://www.meetup.com/DataPhilly/events/230814092/) / [Code for Philly](http://www.meetup.com/Code-for-Philly/events/230802415/) | [MilkBoy](http://www.milkboyphilly.com/) |
 
-On Tuesday July 26, we had a [meetup](https://www.meetup.com/DataPhilly/events/232785717/ "DataPhilly Meetup Details") that split into groups and focused on making sure everyone left with a task they could work on. Thanks [everyone who contributed](https://github.com/cognoma/cognoma/issues/23 "Issue #23: July 19–26 Project Cognoma Acknowledgements") in the week leading up to this event.
+## Maintainers
 
-### July 19
+Cognoma relies on our generous community maintainers. Thanks to the following maintainers for their help:
```

Comment:
Also, it may be helpful to note in text that the number of contributors is far larger than this. If it's possible to track who has participated outside the sandbox and automatically or periodically update a list of names here (github linked to usernames) of all contributors that would be great.